files/default/sql/ddl/updates/undo/3.0.0__undo.sql

```diff
@@ -66,3 +66,5 @@ ALTER TABLE `hopsworks`.`job_alert` DROP FOREIGN KEY `fk_job_alert_1`, DROP COLU
 ALTER TABLE `hopsworks`.`feature_group_alert` DROP FOREIGN KEY `fk_feature_group_alert_1`, DROP COLUMN `receiver`;
 
 DROP TABLE IF EXISTS `hopsworks`.`alert_receiver`;
+
+ALTER TABLE `hopsworks`.`feature_store_statistic` DROP CONSTRAINT `for_transformation`;
```

Comment:
did you mean `DROP COLUMN`?
```suggestion
ALTER TABLE `hopsworks`.`feature_store_statistic` DROP COLUMN `for_transformation`;
```